tracer/test/Datadog.Trace.Tests/Debugger/SymbolExtractor/Approvals/SymbolExtractorTest.Datadog.Trace.Tests.Datadog.Trace.Tests.Debugger.SymbolExtractorTest.verified.txt

```diff
@@ -0,0 +1,309 @@
+﻿{
+  "service": "test",
+  "env": "",
+  "version": "",
+  "language": "dotnet",
+  "scopes": [
+    {
+      "scopeType": "Assembly",
+      "name": "Datadog.Trace.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
+      "type": null,
+      "sourceFile": "C:\\dev\\Datadog\\dd-trace-dotnet\\tracer\\test\\Datadog.Trace.Tests\\bin\\Debug\\net6.0\\Datadog.Trace.Tests.dll",
+      "startLine": -1,
+      "endLine": 2147483647,
+      "languageSpecifics": null,
+      "symbols": null,
+      "scopes": [
+        {
+          "scopeType": "Class",
+          "name": "Datadog.Trace.Tests.Debugger.SymbolExtractorTest",
+          "type": null,
+          "sourceFile": "C:\\dev\\Datadog\\dd-trace-dotnet\\tracer\\test\\Datadog.Trace.Tests\\Debugger\\SymbolExtractorTest.cs",
+          "startLine": 24,
+          "endLine": 90,
+          "languageSpecifics": {
+            "accessModifiers": [
+              "Public"
+            ],
+            "annotations": [
+              "AutoLayout, AnsiClass, Class, Public, BeforeFieldInit"
+            ],
+            "superClasses": [
+              "System.Object"
+            ],
+            "interfaces": null,
+            "returnType": null
+          },
+          "symbols": [
+            {
+              "name": "_extractor",
+              "type": "SymbolExtractor",
+              "symbolType": "Field",
+              "line": -1
+            }
+          ],
+          "scopes": [
+            {
+              "scopeType": "Method",
+              "name": ".ctor",
+              "type": null,
+              "sourceFile": "C:\\dev\\Datadog\\dd-trace-dotnet\\tracer\\test\\Datadog.Trace.Tests\\Debugger\\SymbolExtractorTest.cs",
+              "startLine": 24,
+              "endLine": 27,
+              "languageSpecifics": {
+                "accessModifiers": [
+                  "Public"
+                ],
+                "annotations": [
+                  "PrivateScope, ReuseSlot, Public, HideBySig, SpecialName, RTSpecialName",
+                  "IL"
+                ],
+                "superClasses": null,
+                "interfaces": null,
+                "returnType": "System.Void"
+              },
+              "symbols": [
+                {
+                  "name": "this",
+                  "type": "Datadog.Trace.Tests.Debugger.SymbolExtractorTest",
+                  "symbolType": "Arg",
+                  "line": 24
+                }
```

Comment:
I'm not sure, do we need the `this` arg? Fields of `this` are already "in-scope" because they are defined on the enclosing class.